plugins/gcp-projects/src/api/GcpClient.ts

```diff
@@ -17,6 +17,7 @@
 import { GcpApi } from './GcpApi';
 import { Operation, Project } from './types';
 import { OAuthApi } from '@backstage/core-plugin-api';
+import packageinfo from '../../package.json';
```

Comment:
Could you deconstruct the package info to only access the version here? It avoids bringing the entire package.json into the build. Same thing for the other similar solutions too tbh

Reply:
Hi @Rugvip 

Sure, thank you for the annotation, l pushed the change with a new file named version.json in order to just get version number and not the entire package, let me know if that is what you asked for.

Thank you

Reply:
@Rugvip in this way we not have the error that breaks the front end, but let me know if its ok from your point of view.

Thank you!